cli/src/klio_cli/commands/job/gke.py

```diff
@@ -123,6 +129,26 @@ def _update_deployment(self, replica_count=None, image_tag=None):
         for message in log_messages:
             logging.info(message)
 
+        ui_link = self._build_ui_link_from_current_context()
+        logging.info(f"Deployment details: {ui_link}")
+
+    def _build_ui_link_from_current_context(self):
+        context_name = self.kubernetes_active_context["name"]
+        # context seems to follow the format
+        # gke_{project}_{region}_{clustername}
+        _, _, region, current_cluster = context_name.split("_")
```

Comment:
Sticking to inferring the region for now rather than looking it up in the [GKE API](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters/list), since we have to parse the context anyway to get the name of the cluster.